pkg/plugins/policies/meshhttproute/plugin/v1alpha1/testdata/match-priority.listeners.golden.yaml

```diff
@@ -0,0 +1,71 @@
+resources:
+- name: outbound:127.0.0.1:10001
+  resource:
+    '@type': type.googleapis.com/envoy.config.listener.v3.Listener
+    address:
+      socketAddress:
+        address: 127.0.0.1
+        portValue: 10001
+    filterChains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typedConfig:
+          '@type': type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          httpFilters:
+          - name: envoy.filters.http.router
+            typedConfig:
+              '@type': type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
+          normalizePath: true
+          routeConfig:
+            name: outbound:backend
+            requestHeadersToAdd:
+            - header:
+                key: x-kuma-tags
+                value: '&kuma.io/protocol=http&&kuma.io/service=web&'
+            validateClusters: false
+            virtualHosts:
+            - domains:
+              - '*'
+              name: backend
+              routes:
+              - match:
+                  path: /v1/specific
+                name: yih0Y9HpUXVKBUFYOjdHJ6ltlVzhID6xNw9/5VgD9w8=
+                route:
+                  cluster: backend
+                  timeout: 0s
+              - match:
+                  path: /v1
+                name: bzv64o+joRk33jfDADgGv5ar4QyONO0NEVKnoBfJW/E=
+                route:
+                  cluster: backend
+                  timeout: 0s
+              - match:
+                  prefix: /v1/
```

Comment:
Shouldn't it be higher because it's longer than the one above? I'm uncertain if this applies to the path that we generated.